### PR TITLE
Ternullo Pull Request

### PR DESCRIPTION
**Pull request by sternullo15**
After I forked README.md and opened it into a new project in my R Studio, I looked at the R Markdown "Cheat Sheet" we were provided in class. I wanted to make the largest header the assignment (#), followed by a much smaller due date, then my name and the title of the document, all in various sizes of headers. I decided to add a photo next, and borrowed the link from the UChicago Sociology website. I played around with a caption but it seemed unnecessary given the titles of the document. I started writing a bit about myself and then pressed preview to see what it all looked like. The image showed up just fine, but the text below it was all jumbled together without line spacing. I googled how to add blank spaces in R Markdown, so I added a <br> below the html for the photo and the double space after the lines of text. I added some bold and italics using the asterisks as well. I couldn't quite understand from the cheat sheet how to add bolds and italics (I kept previewing and in each attempt it just erased the text I wanted to emphasize). I googled again and found an appropriate answer. I had another slight struggle embedding the link to my profile. I at first just added the website link, but then realized that I could actually link it to the word. Then I left a space between the word and the link and was confused for a few minutes as to why it wasn't working. That was the last hurdle!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,19 @@
...
+## Stephanie Ternullo
+### Autobiography 2018
+
+![](https://sociology.uchicago.edu/sites/sociology.uchicago.edu/files/styles/columnwidth-wider/public/uploads/images/TERNULLO_STEPHANIE.JPG?itok=TpLZuX2w.png)  
+<br>
+
+My name is Stephanie Ternullo. I am a First-Year PhD Student in the **Sociology Department**, interested in political sociology. I'm planning to use geospatial techniques to model changes in voting outcomes by county in the U.S. from the 1950s to the present. You can find my profile on the Sociology Department [website](https://sociology.uchicago.edu/directory/stephanie-ternullo). I am originally from Connecticut, but I am coming to Chicago most recently from a job in Washington, D.C.
+
+My academic background is a bit varied. In the past, I have studied:  
+1. Economics at Amherst College, *B.A*
```
Comment (soohyun1995) [inline README.md:16]: Good job on using lists!

File: README.md [hunk 2]
```diff
@@ -1,2 +1,19 @@
-# hw01
-Repository for homework 1
+---
+output: html_document
+---
+# Computing for the Social Sciences, HW1
+##### Due: April 2, 2018
+
+## Stephanie Ternullo
```
Comment (soohyun1995) [inline README.md:7]: I like how you used headers here!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,19 @@
...
+### Autobiography 2018
+
+![](https://sociology.uchicago.edu/sites/sociology.uchicago.edu/files/styles/columnwidth-wider/public/uploads/images/TERNULLO_STEPHANIE.JPG?itok=TpLZuX2w.png)  
+<br>
+
+My name is Stephanie Ternullo. I am a First-Year PhD Student in the **Sociology Department**, interested in political sociology. I'm planning to use geospatial techniques to model changes in voting outcomes by county in the U.S. from the 1950s to the present. You can find my profile on the Sociology Department [website](https://sociology.uchicago.edu/directory/stephanie-ternullo). I am originally from Connecticut, but I am coming to Chicago most recently from a job in Washington, D.C.
```
Comment (soohyun1995) [inline README.md:13]: Good job on using italics in the middle of the sentence!

**Review by soohyun1995 — COMMENTED** [review]
Coding style, presentation, and creativity are all excellent! I love how you incorporated headers, lists, italics, and images to your biography! :)

**Review by gmvelez — COMMENTED** [review]
Evaluation HW-01 for Stephanie Ternullo
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |    x       |              |            |

This is a good job with correct use of various formatting techniques, and correctly linking a photo.  Another way that you could have included the photo would have been to save it in the same folder you are working out of, then link it just with the name {i.e. }, and then push it to Git along with the readme.  One other detail is that because the top part wasn't formatted right, the output line was also produced in the document (when it is only meant to be a meta note).

**Comment by sbridgewaters** [comment]
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |       x    |              |            |
| **Coding strategy**         |    x       |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |      x     |              |            |

Remarks:
You did a good job of incorporating the tasks of the assignment but could have utilized Markdown more by breaking your bio into more sections, which would have also allowed you to use text emphasis a bit more.

**Pull request closed without merging**